fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/data/LoanAccountData.java

```diff
@@ -127,6 +126,7 @@ public final class LoanAccountData {
     private final LocalDate interestChargedFromDate;
     private final LocalDate expectedFirstRepaymentOnDate;
     private final Boolean syncDisbursementWithMeeting;
+    private final Boolean disallowExpectedDisbursements;
```

Comment:
Please add this attribute to the Avro schema as well.

Reply:
Can someone explain me that?

Reply:
Done!